tests/model-test/cpp_graph_inference.sh

```diff
@@ -432,7 +439,7 @@ function main() {
                     else
                         real_ctx=$ctx # TODO(Zhenzhong): use same ctx for  chatglm & baichuan
                         [[ "${model}" == "chatglm2" || "${model}" == "chatglm-6b" ||
-                            "${model}" == "baichuan-13b" || "${model}" == "baichuan2-13b" ]] && real_ctx=1300
+                            "${model}" == "baichuan-13b" || "${model}" == "baichuan2-13b" ]] && real_ctx=2048
```

Comment:
why change baichuan real_ctx here ?

Reply:
Zhenzhong's pr can automatically allocate memory, so I want to try to use baichuan2-13b inference with 2012 inputs.